schema.sql

```diff
@@ -6,5 +6,6 @@ name text,
 date_of_birth date,
 escape_attempts int,
 neutered boolean,
-weight_kg decimal
+weight_kg decimal,
+species text
 );
```

Comment:
- [ ] Kindly note that you were supposed to run a spacial query for adding the species column in the animal table and not to change the query that creates the animal table. Hint :point_down: 

```sql
   ALTER TABLE animals ADD species varchar(255);
```

Reply:
Thanks. I have implemented it.